dds/DCPS/XTypes/DynamicTypeSupport.h

```diff
@@ -280,7 +280,14 @@ void DataReaderImpl_T<XTypes::DynamicSample>::dynamic_hook(XTypes::DynamicSample
 
 namespace XTypes {
 
-class OpenDDS_Dcps_Export DynamicDataReaderImpl
+#if defined _MSC_VER && _MSC_VER < 1700
```

Comment:
This is slightly confusing.  Does it not actually need to be exported?

Reply:
Based on what we currently have in the repo, we can get along without it.  So that was the other option - just remove the macro on all builds.
In general we'd like to export the these kind of classes so they can be added to unit tests.

Reply:
Ah, ok.  That makes sense then.